core/pom.xml

```diff
@@ -193,7 +192,12 @@
 		<dependency>
 			<groupId>org.ibissource</groupId>
 			<artifactId>ibis-xerces</artifactId>
-			<version>2.12.0.1</version>
+			<version>2.12.0-xml-schema-1.1.1</version>
+		</dependency>
+		<dependency>
```

Comment:
Where do we need the xpath2 processor, or where is it used?

Reply:
It is used by Xerces, but it does not use any dependency management tool (like gradle or maven) so I had to add it for now. Since we will eventually add it to our repository, I can set the dependency on its pom file, and remove it from our pom.

Reply:
Removed xpath dependency from core.